### DISCUSSION TITLE
Issues sending email from a self hosted instance of supabase

### DISCUSSION THREAD

**Discussion by rsatyan** [supabase/supabase#20502] [Questions]
I have configured the email SMTP server correctly in the .env file and have restarted the docker containers, but when i look in the logs. It's trying to use supabase-mailer as the SMTP host. Not sure where this server is coming from, but any help is appreciated.

**Comment by jonathanlal** [comment]
did you figure this out?

**Comment by praisethemoon** [comment]
Can you share your `.env`? Be sure to mask secrets and such.
